.github/workflows/cirrus-failures.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     if: github.event.check_run.app.name == 'Cirrus CI' && github.event.check_run.conclusion == 'failure' && toJson(github.event.check_run.pull_requests) == '[]'
     runs-on: ubuntu-latest
     steps:
+      - name: Log event
+        run: echo "$EVENT"
+        env:
+          EVENT: ${{ toJson(github.event) }}
       - name: Send
         uses: zulip/github-actions-zulip@35d7ad8e98444f894dcfe1d4e17332581d28ebeb
         with:
```

Comment:
Not sure what you did, it looks like merged rather than rebased. This shouldn't be part of the PR.

Reply:
Otherwise, this looks good. Once this is removed, drop me a note on the PR and we can merge. 

Thanks.